index_ar.html

```diff
@@ -5,6 +5,10 @@
 direction: rtl
 
 pagecontent:
+  install:
+    install: Homebrew ثبّت
```

Comment:
Cool. Done locally. Will give it a push up to the remote here within the next couple of hours.